### DISCUSSION TITLE
I can no longer maintain my products!

### DISCUSSION THREAD

**Discussion by tim-lennon** [community/community#43402] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I can no longer maintain my products. Previously OK, the issue started 2 days ago. Prestashop now issues a gateway error. Help!

**Comment by airtower-luna** [comment]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/PrestaShop/PrestaShop? In that case you should look for support information *in that repository*, usually in the README or a file called SUPPORT. If those don't say anything, opening an issue there is usually good. Mind that a bug report needs to contain all relevant details to get a useful answer. :slightly_smiling_face:
